docs/network/smart-rollup-nodes.md

```diff
@@ -21,20 +21,12 @@ You can use a public layer 1 node or set up your own as described in [Installing
 
 ## Running the Smart Rollup node
 
-You can start the Smart Rollup node with a snapshot of the Etherlink state or from Etherlink genesis.
-These instructions cover starting from Etherlink genesis, which requires a recent build of the Octez suite.
-They result in an archive node, which contains a copy of the state at each level since Etherlink genesis.
+You can start the Smart Rollup node with a snapshot of the Etherlink state to prevent it from having to compute the state from Etherlink genesis.
 
-For simplicity, these steps show how to run the octez smart rollup node in observer mode:
+For simplicity, these steps show how to run the smart rollup node in observer mode:
 
-1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`, in one of these ways:
-
-   - To set up a Smart Rollup node from Etherlink genesis, it's best to use recent Octez source code or the latest release of Octez.
-   For more information about building Octez binaries, see the Octez documentation: https://tezos.gitlab.io/index.html
-
-   - Octez binary builds for Linux systems are available for the [x86_64](https://gitlab.com/tezos/tezos/-/jobs/6849203975/artifacts/browse/octez-binaries/x86_64/) architecture.
-
-   - Docker images are available at https://hub.docker.com/r/tezos/tezos.
+1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`.
+Until Octez version 20 is released, the best place to get binary files to use with Etherlink is https://gitlab.com/nomadic-labs/tezos/-/releases.
```

Comment:
Sadly won't be part of `v20` so let's say "future octez release" or something along those lines.

Reply:
> The best place to get the most recent binary files to use with Etherlink is https://gitlab.com/nomadic-labs/tezos/-/releases.